Twig/Extension/FormTypeExtension.php

```diff
@@ -23,7 +23,7 @@ class FormTypeExtension extends \Twig_Extension
 
     public function __construct($formType)
     {
-        $this->wrapFieldsWithAddons = ($formType == 'standard');
+        $this->wrapFieldsWithAddons = $formType == 'standard';
```

Comment:
This is not related to the PR. Submit another one with an explanation of why this modification.